src/api/spec/factories/packages.rb

```diff
@@ -20,5 +20,12 @@
         end
       end
     end
+    factory :package_with_service do
+      after(:create) do |package|
+        # NOTE: Enable global write through when writing new VCR cassetes.
+        # ensure the backend knows the project
+        Suse::Backend.put("/source/#{CGI.escape(package.project.name)}/#{CGI.escape(package.name)}/_service", '<service/>')
```

Comment:
This should actually be URI.escape - but I see yours is not the first spec to mix them ;(